implementer-guide/setup/how-to-use-azure-ad-to-login-to-trs.md

```diff
@@ -1,104 +1,138 @@
 
 ![AzureAD - application setup](../../images/report-server-images/HowToLoginUsingADFS/azure_ad_step4.png)
 
+This concludes the configuration of the Azure application. We have obtained all the necessary information to register it in the Telerik Report Server web application.
 
-### 6.   Getting the Application (client) ID
+### 6.  Configure Telerik Report Server instance
 
-After registration is complete, go to **Overview** tab and get the **Application (client) ID**. It must be pasted in the ClientID field the Report Server's Authentication Settings page.
+In Azure, navigate to the Overview tab and click on the Endpoints toolbar button.
 
 ![AzureAD - application setup](../../images/report-server-images/HowToLoginUsingADFS/azure_ad_step5.png)
 
+Copy the OAuth 2.0 authorization endpoint (v2). It should be the first one on the list, as shown in the screenshot below.
 
-### 7.  Setting Endpoints
+![AzureAD - application setup](../../images/report-server-images/HowToLoginUsingADFS/azure_ad_step6.png)
 
-Go to **Endpoints** tab and copy the OAuth 2.0 authorization endpoint (v2). It should be the first one in the list, as shown on the screenshot below. Paste it into **Authority field** in the Report Server's Authentication Settings page.
+Open Telerik Report Server Manager web application and log in with a user with administrative permissions. Navigate to Configuration and click on the Authentication tab.
+- Check the **Enabled** checkbox below the Federation provider label.
+- Paste the copied OAuth 2.0 authorization endpoint URL into the **Authority** textbox.
+- Switch again to the Endpoints tab in Azure and copy the **Federation metadata document** URI.
+- Switch to Report Server's Configuration page and paste it into the **Metadata URI** textbox. Append the following chunk to the URI: ````?appid={your-application-id}```` where ````{your-application-id}```` is the **Application (client) ID** of the Azure application. You can see it in the Essentials panel in the Overview tab. Finally, the contents of the **Metadata URI** textbox should look like this: 
+````https://login.microsoftonline.com/{your-directory(tenant)-id}/federationmetadata/2007-06/federationmetadata.xml?appid={your-application-id}````
+- Enter the URL of the Telerik Report Server application in the **Relying Party ID** textbox. Its value must be the same as the value of the Redirect URI configured in step 5.
 
-![AzureAD - application setup](../../images/report-server-images/HowToLoginUsingADFS/azure_ad_step6.png)
+With this step the authentication configurations for both the Azure application and Telerik Report Server are complete. At this point, you should be able to log in to Report Server using the web interface, by clicking on the blue button "**Active Directory Credentials**" on the login screen.
```

Comment:
"At this point..." - this leaves the impression, that we are good to log in, but we still have no users. Maybe rephrase the sentence that we are good to go once we add the users to RS.

Reply:
Correct, I assumed the users are already added to RS, which might not be true. I changed it to:
The configuration of the Azure application and Telerik Report Server is now complete. In order to be successfully logged-in in Report Server Manager web app, ensure that the Azure AD users are added as Telerik Report Server users, as described in section ![**Report Server Users**](./how-to-use-azure-ad-to-login-to-trs#report-server-users) below.